docs/makeup-hoist-element/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Makeup Hoist Element</title>
+    </head>
+    <body>
+        <p>This package hoists an element directly below document.body. This is useful for various accessibility purposes.</p>
+
+        outer
+        <div>
+            first child
+            <div>
+                second child
+                <div id="hoist-this">
```

Comment:
Can you add one or two more hoistable element examples to the page so that we know the singleton pattern is working (i.e. we should see the currently hoisted element get unhoisted). I suggest adding an example of the edge case where an element is already a direct descendant of the body.